ui/src/me/components/LogoutButton.tsx

```diff
@@ -4,23 +4,11 @@ import {Link} from 'react-router'
 
 // Components
 import {Button, ComponentSize} from '@influxdata/clockface'
-import CloudExclude from 'src/shared/components/cloud/CloudExclude'
-import CloudOnly from 'src/shared/components/cloud/CloudOnly'
-import {CLOUD_URL, CLOUD_LOGOUT_PATH} from 'src/shared/constants'
 
 const LogoutButton: SFC = () => (
-  <>
-    <CloudExclude>
-      <Link to="/logout">
-        <Button text="Logout" size={ComponentSize.ExtraSmall} />
-      </Link>
-    </CloudExclude>
-    <CloudOnly>
-      <a href={`${CLOUD_URL}${CLOUD_LOGOUT_PATH}`}>
-        <Button text="Logout" size={ComponentSize.ExtraSmall} />
-      </a>
-    </CloudOnly>
-  </>
```

Comment:
My thinking in removing these lines is the fact that the `Logout` Component that is being rendered on `/logout` is already performing this cloud logic here:

https://github.com/influxdata/influxdb/blob/master/ui/src/Logout.tsx#L30-L33

Since we want to ensure that this is actually running:

https://github.com/influxdata/influxdb/blob/master/ui/src/Logout.tsx#L21-L29

It is imperative that the `Logout` component be rendered in one way or another